openfisca_us/tests/policy/baseline/gov/states/ca/tax/income/deductions/standard/ca_standard_deduction

```diff
@@ -0,0 +1,17 @@
+- name: Single filer 
+  period: 2021
+  input:
+    state_code: CA
+    adjusted_gross_income: 1
+    filing_status: SINGLE
+  output:
+    ca_renters_credit: 4_803
+
+- name: Joint filer 
+  period: 2021
+  input:
+    state_code: CA
+    adjusted_gross_income: 1
+    filing_status: JOINT
+  output:
+    ca_renters_credit: 9_606
```

Comment:
Could you run `make format` so we avoid this unterminated line? If you get an error with `linecheck`, you might need to install the dev dependencies (`pip install -e .[dev]` from the root of the repo)

Reply:
Done